pkg/cli/cvd.go

```diff
@@ -226,9 +229,10 @@ func listCVDs(c *cobra.Command, flags *ListCVDsFlags, opts *subCommandOpts) erro
 	var wg sync.WaitGroup
 	var mu sync.Mutex
 	var merr error
-	for _, host := range hosts {
+	sortedWriter := newSortedWriter(c.OutOrStdout())
```

Comment:
two comments in one:
1- (nit) Don't like the overly complex use of mutexes and wait groups
2- Possible high latency from waiting for all calls to succeed before printing anything.

Both can be addressed by creating a channel per routine and having each routine write the result (message or error) to it. Then once all routines have been started the main routine reads from each channel in the right order and prints accordingly.

Reply:
great point, rephrased with channels.